src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NotYetReady/Count.txt

```diff
@@ -0,0 +1,145 @@
+﻿// ******** TABLE WITH NO BLANK RECORDS ********
+
+>> Count([0])
+1
+
+>> Count([1])
+1
+
+>> Count(Table({a:1},{a:2},{a:3}))
+3
+
+﻿>> Count([1, 2, 3, 4, 5])
+5
+
+// ******** TABLE WITH FEW BLANK RECORDS ********
+
+﻿>> Count([0,""])
+1
+
+>> Count([1,2,3,4,Blank()])
+4
+
+>> Count([1,2,Blank(),Blank(),Blank()])
+2
+
+>> Count([1,2,3,If(1<0,4),Blank()])
+3
+
+>> Count(Table({a:1},{a:Blank()},{a:2},{a:Blank()} ))
+2
+
+>> Count(Table({a:1},{a:2},If(1<0,{a:3}),If(1>0,{a:4}),{a:5}))
+4
+
+>> Count(LastN([1,2,3,4], Blank()))
+Blank()
+
+>> Count([1,"1","2",Blank()])
+3
+
+//One blank record
+>> Count([Blank()])
+0
+
+//All blank records
+﻿>> Count([Blank(),Blank(),Blank(),Blank(),Blank()])
+0
+
+>> Count(Table({a:Blank()},{a:Blank()},{a:Blank()}))
+0
+
+// ******** ALL DATATPYE RECORDS ********
+
+>> Count([1,true,false,Date(2022,12,12),Time(6,30,30),DateTimeValue("5/12/2022 6:30:30 PM"),"abc"])

```

Comment:
>[1,true,false,Date(2022,12,12),Time(6,30,30),DateTimeValue("5/12/2022 6:30:30 PM"),"abc"] [](http://example.com/codeflow?start=9&length=89)

This doesn't test the count function with all data types: both of them are testing a table of numbers. Ideally we should have tests for tables of all data types (i.e., table of numbers, table of booleans, table of dates, table of date/time values, table of times, table of strings, table of records, table of tables, and so on.

Reply:
Nevermind, Count can only take a table with a single numeric column